NEWS.md

```diff
@@ -1,4 +1,10 @@
 ## formatters 0.4.1.9001
+ * font_size is now 12 across the board, whereas before it was inconsistent between 8 and 12
+ * margins are now correctly interpreted as inches when specified (including the default)
```

Comment:
```suggestion
 * Margins are now interpreted as inches when specified (including the default).
```
again GitHub actions

Reply:
This one is actually wrong and we should not make the change.

Reply:
hi @gmbecker , can you update the correct wording here. Thanks

Reply:
I do not understand what you mean. the wording is correct before the change. the 'now correctly' is there because they were documented as inches before but interpreted as lines in at least some places.

Reply:
I see. Thanks @gmbecker , ok, we can mark this one as resolved then.